packages/react-search-ui-views/src/__tests__/SearchBox.test.tsx

```diff
@@ -105,19 +115,9 @@ describe("inputView prop", () => {
       />
     );
 
-    input = wrapper
```

Comment:
did not make a difference to snapshot so just only removed the incorrect argument for dive.